addons/portal/static/src/scss/portal.scss

```diff
@@ -116,6 +116,13 @@ li > p {
             clear: both;
         }
     }
+    .navbar %container-flex-properties {
+        // Bootstrap set-up flex alignment to "space-between" here. The hack made
+        // above to restore the BS3 protection has to be disabled.
+        &::before, &::after {
+            display: none;
+        }
+    }
```

Comment:
@Brieuc-brd What do you think? Do you agree this should be the solution instead of the specific one you made in blog?

Reply:
I agree, it sounds good :)